src/panels/ConfigPanel/_config-panel.scss

```diff
@@ -10,31 +10,12 @@ $panel-padding: 1.5rem;
 }
 
 .config-panel {
-  $magic-title-height: 4.7rem;
```

Comment:
This component is now wrapped in the new `Panel` component instead of using `SlidePanel` so there were some styling changes needed.